crates/dpapi/README.md

```diff
@@ -0,0 +1,9 @@
+# dpapi-rs
+
+This crate contains the Windows [DPAPI](https://learn.microsoft.com/en-us/windows/win32/seccng/cng-dpapi) implementation. It can encrypt the data/decrypt DPAPI blobs using the domain's root key.
```

Comment:
```suggestion
This crate contains a Windows [DPAPI](https://learn.microsoft.com/en-us/windows/win32/seccng/cng-dpapi) implementation. It can encrypt the data/decrypt DPAPI blobs using the domain's root key.
```

Reply:
Done